notebooks/6_search_in_continuous_spaces/index.md

```diff
@@ -0,0 +1,313 @@
+﻿
+Search in Continuous Space
+#
+# **Table of Contents**

```

Comment:
Try to use the same format throughout this part, if you start your words with uppercase letters then do the same for other words.